ci/run_tests.sh

```diff
@@ -16,12 +16,17 @@ fi
 
 echo "running tests with flags: ${test_flags[*]} and features: ${test_features}"
 
+# Event listener tests have to be run in their own `wasm-pack test` invocation or they won't work.
```

Comment:
Heads up: these bash scripts will be removed entirely in #1541. You don't need to switch to cargo make just yet because there are still some changes incoming but be prepared to update once the aforementioned PR lands.

Reply:
#1541 just landed. Can you update this to work with cargo-make?
Let me know if you need help with it.